### DISCUSSION TITLE
How do I change Uploads folder to a different folder?

### DISCUSSION THREAD

**Discussion by tablifiedapps** [paukstelis/octoprint_deploy#43] [Q&A]
I have multiple instances of octoprint running off my Linux server installed using octoprintdeploy. I would like each instance to access a specific folder on my internal hdd for gcode files. Is this possible? Where can I change the path?

**Comment by paukstelis** [comment]
Not sure exactly what you are referring to. They all do have their own folders by default: `~/.instancename/uploads`
If you want gcode to be accessible for all instances, you can just change the Folder setting to be the same for all instances (Settings->OctoPrint->Folders).

**Reply by tablifiedapps** [reply]
I want one central folder that all the instances have access to. I dont want each one to have its own folder.  Where is the folder setting I am not sure what you are referring to.

**Reply by paukstelis** [reply]
Then simply pick one instances that will have all your files and change the upload folder of all the other instances to have the same path.
![image](https://user-images.githubusercontent.com/28326982/203855265-8d171c5c-f03b-41ba-b3ef-fe70bf74ffc0.png)
You should only need to change the upload folder unless you are doing some other specialized things.
